todo_app/app.py

```diff
@@ -1,14 +1,45 @@
-from flask import Flask
+from flask import Flask, request, redirect, render_template
 
 from todo_app.flask_config import Config
+from todo_app.data import session_items as session
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['POST', 'GET'])
 def index():
-    return 'Hello World!'
+    
+    if request.method == 'POST':
+        item_title = request.form['Title']
+        session.add_item(item_title)
+        return redirect('/')
```

Comment:
This could probably be its own route. POST to '/' isn't particularly meaningful.